ansible/testbed-cli.sh

```
@@ -810,6 +815,25 @@ function deploy_topo_with_cache
+  inventory=$2
+  passfile=$3
+  shift
+  shift
+  shift
+
+  echo "Configuring testbed '$testbed_name' as a virtual chassis"
+
+  read_file $testbed_name
+
+  ansible-playbook -i "$inventory" testbed_config_vchassis.yml --vault-password-file="$passfile" -l "$duts" -e testbed_name="$testbed_name" \
+  -e topo="$topo" -e testbed_file=$tbfile -e vm_file=$vmfile -e deploy=true $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)